views/js/controller/routes.js

```diff
@@ -0,0 +1,12 @@
+define({
+    'Monitor' : {
+        'actions' : {
+            'index' : 'controller/monitoring'
```

Comment:
Yes, that is what I did in first place. But the compilation just provided a very small minified file that only contains address of the target controllers. Is this ok?